Tests/Functional/Fixtures/app/config/config.yml

```diff
@@ -5,6 +5,8 @@ framework:
   test: ~
   session:
     storage_id:  session.test_storage
+  templating: 
+    engines: ['php']
```

Comment:
can you add a comment that this is just to work around a bug in symfony 2.7 (and check if there is a report, or create a report? and then link to that.) so that we remove this again when no longer needed.